.travis.yml

```diff
@@ -183,15 +183,11 @@ matrix:
 script:
   - |
     if [ "$TEST_HARNESS" == true ]; then
-      make test-runner
       export DISPLAY=:1
       Xvfb :1 -screen 0 1024x768x24 &
       xfwm4 &
       export ASAN_OPTIONS=detect_leaks=0;
-      if ! ./test-runner ; then
-        travis_terminate 1
-      fi
-      ./CommandLine/testing/GitHub-ImageDiff.sh  
+      ./ci-regression.sh "/tmp/enigma-master"
```

Comment:
Thanks, I think your clarification here really clears a lot of things up. It also justifies that we loop over a directory and report arbitrary differences in the output of the regression tests now, instead of asserting and failing on golden data like fundies first had us doing.

I think it's not as egregious considering how easy it is to run them locally, as fundies does, by calling `ci-regression.sh` so I am not so much concerned about any misnomers etc. As long as new contributors can easily run this apparatus and make sense of what is reported to them.